lte/gateway/c/core/oai/tasks/mme_app/mme_app_state_manager.cpp

```diff
@@ -30,7 +30,7 @@ constexpr char MME_NAS_STATE_KEY[] = "mme_nas_state";
 const int NUM_MAX_UE_HTBL_LISTS = 6;
 constexpr char UE_ID_UE_CTXT_TABLE_NAME[] =
     "mme_app_mme_ue_s1ap_id_ue_context_htbl";
-constexpr char IMSI_UE_ID_TABLE_NAME[] = "mme_app_imsi_ue_context_htbl";
+constexpr char MME_IMSI2MME_UE_ID[] = "mme_imsi2ue_id_map";
```

Comment:
Change the name to "MME_IMSI2MME_UE_ID_MAP_NAME" to indicate it's map and it's name